### PR TITLE
Added support Cyrillic symbols in description attributes

### PR DESCRIPTION
**Pull request by anykeysas**
Added support Cyrillic symbols in description attributes in metadataPage
servicestack
s

### CODE REVIEW

File: src/ServiceStack/Metadata/BaseMetadataHandler.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public abstract class BaseMetadataHandler : HttpHandlerBase
         public override void Execute(HttpContextBase context)
         {
             var writer = new HtmlTextWriter(context.Response.Output);
-            context.Response.ContentType = "text/html";
+            context.Response.ContentType = "text/html;utf-8";
```
Comment (mythz) [inline src/ServiceStack/Metadata/BaseMetadataHandler.cs]: I'd expect this to also be `text/html; charset=utf-8`

**Comment by mythz** [comment]
Thanks for the PR, can you also [approve the Contributor License Agreement](https://docs.google.com/forms/d/16Op0fmKaqYtxGL4sg7w_g-cXXyCoWjzppgkuqzOeKyk/viewform) so we can merge this, thx.

**Comment by anykeysas** [comment]
Done

**Comment by mythz** [comment]
Cheers :+1:

**Pull request merged**